rpki-validator/src/main/java/net/ripe/rpki/validator3/domain/validation/CertificateTreeValidationService.java

```diff
@@ -341,6 +343,9 @@ private CertificateAuthorityValidationResult validateCertificateAuthority(
                 return result;
             }
 
+            long nextUpdateTimeDifferenceMs = manifest.getNextUpdateTime().getMillis() - x509Crl.getNextUpdateTime().getMillis();
```

Comment:
Where does the requirement come from that they have the same timestamp?

I feel like warning earlier & rejecting (with a higher threshold) may be the right thing to do here. But that would require us to run this for a while.

Reply:
https://tools.ietf.org/html/draft-ietf-sidrops-6486bis-03#section-4.2.1 under `nextUpdate`:

> If a manifest encompasses a CRL, the nextUpdate field of the manifest MUST match that of the CRL's nextUpdate field, as the manifest will be re-issued when a new CRL is published.

There is also a requirement that a single-use EE certificate should have a matching validity period, but I do not think we can distinguish between single-use and sequential-use EE certificates as RP.